modules/nf-core/canu/meta.yml

```
@@ -0,0 +1,72 @@
+name: "canu"
+description: Accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads.
+keywords:
+  - Assembly
+  - pacbio
+  - hifi
+  - nanopore
+tools:
+  - "canu":
+      description: "Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing."
+      homepage: "https://canu.readthedocs.io/en/latest/index.html#"
+      documentation: "https://canu.readthedocs.io/en/latest/tutorial.html"
+      tool_dev_url: "https://github.com/marbl/canu"
+      doi: "doi:10.1101/gr.215087.116"
+      licence: "['GPL v2 and others']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
```

That code is in the template but misleading I think, because canu is for single end reads only, correct? If yes:
```suggestion
        e.g. [ id:'test', single_end:true ]
```